common/genes/wc_gene_body_naga_test.txt

```diff
@@ -45,6 +45,36 @@ morph_genes = {
 					}
 					priority = 1
 				}
+				decal = {								
```

Comment:
Can we do this for scales as well on the back and back of head?

Reply:
> Can we do this for scales as well on the back and back of head?

Which areas?

Reply:
May be unecessary. In WoW you end up with 4 colors: belly, skin, scales and webbing.
![image](https://user-images.githubusercontent.com/46745564/127218739-de8fff1a-2a8a-452d-9399-6bb80920a7ba.png)
Like here, webbing pink, scales green, belly red, skin orange. If its not doable then its fine.